ext/exif/tests/gh16409.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+GH-16409 (Segfault in exif_thumbnail when not dealing with a real file)
+--EXTENSIONS--
+exif
+--SKIPIF--
+<?php
+if (PHP_OS_FAMILY === "Windows") die("skip *nix only");
```

Comment:
Why? The test succeeds for me on Windows (ZTS).

Reply:
Didn't expect popen to work on Windows, but to be honest I just assumed it and I didn't try it as I want to avoid Windows unless I really have to.

Reply:
My working test was actually:

````php
<?php

$p = popen("echo 1", "r");
exif_thumbnail($p);
````
:)